Update CHANGELOG

rexmac opened the pull request:
Prep for v2.4.0 release

ismail-syed commented on CHANGELOG.md:
```diff
@@ -1,6 +1,8 @@
 # Changelog
 
-## [Unreleased]
+[//]: # '## [Unreleased]'
```
I thought we generally did the following for all our readmes. At least that's what I've noticed. 

```
<!--## [Unreleased] -->
```

rexmac replied:
🤷‍♂️ I was just reverting to what it was before.

I definitely prefer the `<!--` syntax to the archaic `[//]: #`.

rexmac commented:
@ismail-syed I've reverted the version bump and will bump and tag using `yarn version` as you suggested. I've also changed the `Unreleased` comment syntax. :wink:

nimzco commented:
🎉
